cabal.project

```diff
@@ -3,9 +3,12 @@
 
 packages: .
           ./liquid-fixpoint
+          ./text-format
```

Comment:
https://github.com/bos/text-format/pull/21 local.

Let's see what Hackage Trustees can do with that.